<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogShowStrongs</class>
 <widget class="QDialog" name="DialogShowStrongs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>396</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>353</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>753</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Strong's Definition</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>355</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>45</y>
     <width>76</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLanguage">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>40</y>
     <width>56</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLeft">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRight">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLast">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>40</y>
     <width>65</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; Last</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPictures">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Pictures</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDictionary">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Dictionary</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>25</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBibleDictionary">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>130</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Bible Dictionary</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
